### ISSUE TITLE
Подключение к датчику расстояния

### ISSUE THREAD

**Issue by RodionGork** [el-pths/theremin-ok#4]
**Дедлайн: 2016-10-09**

После того как датчик выбранный в #3 будет доступен, написать программу для контроллера, осуществляющую измерение и выводящую результат через последовательный порт, так чтобы можно было убедиться в работоспособности датчика и оценить его точность.

_UPD_

Итак, выбран датчик `US-015` (или аналог `HC-SR04`). Нужно разобраться как с ним работать. Для этого дана таинственная картинка ниже. Также рекомендуется прочесть [это описание](http://amperka.ru/product/hc-sr04-ultrasonic-sensor-distance-module), параграф "Распиновка".

![Сигналы датчика US-015](https://raw.githubusercontent.com/el-pths/theremin-ok/master/aux/us-015-signal.jpg)

После того как станет понятно что изображено на картинке, какие сигналы подаются на датчик и принимаются с него, нужно описать (здесь в комментарии, словами) этот несложный алгоритм. После того как описание будет проверено, пишем программу работающую так:
- запросить и получить значение с датчика;
- вывести его через последовательный канал (через `Serial.println`);
- если расстояние меньше 10 см, включить красный светодиод, а если меньше 20см - зеленый;
- подождать 1 секунду, потом повторить.

**Comment by oliveoilkk** [comment]
Так, ну, на вход приходит логическая единица, потом, я так понимаю, он высылает вперед 8 звуковых импульсов, а затем, обработав вернувшиеся импульсы, выводит единицу на время, пропорциональное расстоянию.

**Comment by RodionGork** [comment]
Суть отражена верно. Предлагаю продолжать!

**Comment by oliveoilkk** [comment]
Done!
https://github.com/el-pths/theremin-ok/blob/master/DistanceSensor.ino

**Comment by RodionGork** [comment]
Much better, thanks!
